README.md

```diff
@@ -15,26 +15,25 @@ cargo build --release
 ```
 
 ## Running
-Chombot requires a few settings passed via environment variables. These are:
-* `DISCORD_TOKEN` - the Discord bot token. You can obtain one by [creating a Discord app](https://discord.com/developers/applications). Make sure to copy your Bot token, not a client secret.
-* `GUILD_ID` - your guild ID.
-* `FEATURE_TOURNAMENTS_WATCHER` - `true`, if you want to receive the notification about EMA tournament updates.
-* `TOURNAMENTS_WATCHER_CHANNEL_ID` - ID of the channel used for notifications about EMA tournament updates.
+Chombot requires a few settings passed via the `config.toml` file that must be present in the working directory. Note that this file serves as a persistence storage for the bot's state, so the bot process must have write permissions for it. These are:
```

Comment:
I am unsure if user configuration and the bot's state storage being the same file is the best idea. I see points for both sides; I just thought it might be worth pointing it out.

Reply:
Actually, I think you're right. It wouldn't be nice if the user data wiped the discord token, too in case of some kind of failure. Let me split these two guys...

Reply:
Done!